packages/charts/react-charting/src/components/DeclarativeChart/PlotlySchemaAdapter.ts

```diff
@@ -549,7 +551,10 @@ export const transformPlotlyJsonToScatterChartProps = (
       roundedTicks: true,
       yMinValue: yMinMaxValues.startValue,
       yMaxValue: yMinMaxValues.endValue,
+      width: input.layout?.width,
+      height: input.layout?.height ?? 350,
       hideTickOverlap: true,
+      enableReflow: false,
```

Comment:
>enableReflow: false, [](http://example.com/codeflow?start=6&length=20)

whyis enable reflow false in this case

Reply:
In LineChart, enableReflow is set to true by default. But it doesn't work well with the hideTickOverlap prop, so I disabled reflow here.